lms/envs/common.py

```diff
@@ -2333,6 +2333,7 @@ def _make_locale_paths(settings):
     'openedx.features.portfolio_project',
     'openedx.features.course_duration_limits',
     'openedx.features.content_type_gating',
+    'openedx.features.subscriptions',
```

Comment:
We should make `subscription` a [plugin](https://github.com/edx/edx-platform/blob/master/docs/guides/extension_points.rst#platform-features-integrating-python-code) 
https://github.com/edx/edx-django-utils/blob/master/edx_django_utils/plugins/README.rst

Reply:
As discussed, for now it will be very difficult to create a separate subscription plugin due to subscription code's dependability on core ECOM, LMS code. We will discuss and achieve this requirement in a separate story.